CLAUDE.md

```
@@ -199,6 +196,52 @@ supabase db push --linked      # Push migrations to production
+# 3. Review generated SQL in supabase/migrations/
+
+# 4. Apply to local database
+pnpm db:migrate:local    # Uses: supabase migration up --local
+
+# 5. Test your changes locally
+
+# 6. Apply to production (after PR merge)
+pnpm prod:db:migrate     # Uses: supabase db push --linked
+```
+
+2. **For Rapid Prototyping Only** (bypasses migration files):
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)